kong/clustering/rpc/concentrator.lua

```diff
@@ -270,7 +342,12 @@ end
 
 
 -- enqueue a RPC response from CP worker with ID worker_id
-function _M:_enqueue_rpc_response(worker_id, payload)
+function _M:_enqueue_rpc_response(worker_id, payload, collection)
```

Comment:
My understanding is that the `collection` parameter is only used for batch calls? Could you add some comments to explain its usage?

Reply:
added